stafl-devcontainer/user-setup-scripts/setup-all.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# This script should be run upon first user login
+
+# Docker containers derived from stafl-devcontainer may append to this script to install user setup actions.
+
+# correct line endings
+find -path '/user-setup-scripts/*' | xargs dos2unix
```

Comment:
Is this necessary if it's already done in the docker build?

Reply:
ah yes. I thought this would be a more reliable place to put it, so that scripts added after the build of this image are also covered.

forgot to remove the call in the Dockerfile.